### DISCUSSION TITLE
pkg/ccl/workloadccl/allccl : test fails

### DISCUSSION THREAD

**Discussion by gitssinghal** [cockroachdb/cockroach#62438] [Q&A] [answered]
Hi There,

I am running cockroach tests in below environment and allccl test is failing.
cockroach - 20.1.0
go version - 1.15.3
machine : ppc64le

The error it produces is 
    --- FAIL: TestDeterministicInitialData/tpch (12.80s)
        all_test.go:289: 
            	Error Trace:	all_test.go:289
            	Error:      	Not equal: 
            	            	expected: 0xdd952207e22aa577
            	            	actual  : 0x573e192d82e5768a
            	Test:       	TestDeterministicInitialData/tpch

Any pointer in this regard is much appreciated.

Regards,
Swati Singhal

**Comment by knz** [comment, marked as answer]
This version of go is not recmended. Minimum we support is 1.15.10

The error is likely due to the PPC target not generating certain values in the same way.

However really this is a regular issue, and should be filed as such not in the discussion section. Also ping me on the new issue thanks.
